src/internal/sbom/viewer/template.gohtml

```diff
@@ -0,0 +1,264 @@
C87.814 132.467 87.8453 132.514 87.8763 132.56C89.3015 134.676 90.3325 136.207 90.3812 136.957C90.4907 138.588 87.9519 141.354 86.3987 142.925C87.8386 142.728 89.1852 142.1 90.2617 141.125L91.3569 140.051C91.3569 141.861 91.4266 143.672 91.4665 145.482C94.1248 144.189 95.3494 140.24 95.5884 140.2C96.2754 141.423 97.5796 143.512 98.2169 144.736C99.4714 142.388 98.4359 138.936 98.4359 138.926L98.2268 138.21L102.886 137.434L98.1073 134.45C98.0177 134.311 97.9182 134.171 97.8186 134.042L94.533 120.115V119.827L96.4446 111.053C97.1814 113.51 96.0563 117.022 95.8572 120.006H95.9169ZM152.668 79.8071C152.671 79.8088 152.674 79.8104 152.678 79.812C155.048 81.0242 166.287 85.9756 180.327 79.2501C179.072 77.2605 174.353 70.8641 164.725 66.5468C161.196 67.0812 157.593 66.8677 154.152 65.9201C154.411 70.5999 153.914 75.2908 152.678 79.812C152.674 79.8237 152.671 79.8353 152.668 79.8469V79.8071ZM154.062 64.5075C156.72 65.4128 172.889 69.8097 189.019 49.0786C187.027 47.9446 179.819 44.5922 166.677 45.4477V44.9503C161.7 47.536 156.418 49.4878 150.956 50.7598C152.229 54.0151 153.137 57.4015 153.664 60.8567C153.853 62.0703 153.992 63.2939 154.092 64.5075H154.062ZM150.428 49.5263C182.219 41.8864 185.444 23.881 186.181 19.6831C155.316 18.6883 142.971 32.3465 140.631 35.3905C144.861 39.3447 148.201 44.1532 150.428 49.4964V49.5263ZM34.1876 66.1489C43.3076 67.4222 54.0107 66.318 65.4804 60.0808C66.247 61.0159 67.0037 61.8415 67.4717 62.3489C66.1276 63.5625 62.8021 66.885 63.7779 69.3123C63.7978 69.4342 63.7978 69.5585 63.7779 69.6804C63.7779 70.9636 62.7822 73.6594 60.2235 76.8626C57.6647 80.0658 53.254 83.637 46.3941 86.4323C39.2952 89.3271 29.548 91.4062 16.2961 91.3167C17.9687 87.3376 25.1174 71.4212 34.1478 66.1887L34.1876 66.1489ZM66.6851 49.775C66.6818 49.7718 66.6785 49.7687 66.6752 49.7655C59.2877 42.7566 49.5866 38.6938 39.4048 38.345C26.7801 29.7203 23.2356 12.2024 22.5885 8.19345C60.1339 12.2621 69.0946 32.3266 69.1244 32.3764L69.443 33.0628L70.0304 32.6549C80.1598 26.1008 92.0984 22.902 104.151 23.5129C122.411 24.3983 136.459 33.3612 139.317 35.9874C146.694 42.5131 150.896 51.7545 152.34 61.0059C153.903 71.1128 152.15 81.2695 147.421 88.0638C144.123 92.7114 139.724 96.4707 134.617 99.0063C125.338 103.731 113.808 104.975 102.478 103.672C91.1479 102.369 80.0465 98.3696 71.6135 92.4805C66.6178 89.1791 62.4703 84.7489 59.5066 79.5485C60.1312 78.9162 60.7228 78.2522 61.2788 77.559C64.0566 74.2464 65.0523 71.1427 65.0523 69.6505C65.0611 69.3558 65.0138 69.0621 64.9129 68.7851C64.0666 66.7159 68.7361 62.9159 68.7461 62.906L69.3136 62.4484L68.8058 61.9311C68.8058 61.9311 64.5445 57.5441 64.0566 55.5247C63.6484 53.7242 69.1941 49.2975 69.214 49.2875L69.5924 48.9791L69.4132 48.5215C69.4132 48.5215 67.85 44.6917 68.2084 43.3388C68.5669 41.9859 73.9035 40.2948 73.9134 40.2948C70.4884 38.723 67.0336 42.6026 66.944 43.0105C66.5756 44.443 67.6011 47.4273 68.0292 48.5613C67.7125 48.8283 67.2286 49.2427 66.6752 49.7655C66.6719 49.7687 66.6685 49.7718 66.6652 49.775H66.6851Z"
```

Comment:
out of curiosity, how was this file built/generated?

Reply:
what do you mean? It's just plain html.  You mean the embedded SVG?